scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -453,3 +453,98 @@ task PostprocessGermlineCNVCalls {
         File genotyped_segments_vcf = genotyped_segments_vcf_filename
     }
 }
+
+task CollectSampleQualityMetrics {
+    Array[File] genotyped_segments_vcf
+    Array[String] entity_ids
+
+    Int? maximum_number_events = 120
```

Comment:
For the general use, I think this should be a required argument with no default value, as it will depend heavily on experimental design. 

@ldgauthier For production, is it typical to provide PASS/FAIL rather reporting the raw metric and letting users interpret it?

Reply:
That's a fair point about experimental design, especially when it comes to exome interval lists.  For a small panel 60 events might still be outrageous.  I would be more comfortable with a default value if there was a way to tie the maximum event number to the interval list.  Otherwise I guess we can provide recommendations in a readme somewhere.

This is a little different from production, but we do have some hard pass/fail cutoffs, though those are things like coverage, contamination, and percent chimeric reads, which won't vary based on capture.

Reply:
Okay I moved the `maximum_number_events` argument up to the workflow level input